manim/mobject/three_dimensions.py

```diff
@@ -254,6 +260,106 @@ def param_surface(u, v):
 # Specific shapes
 
 
+class Plane(ParametricSurface):
+    """
+    A 3D Plane Mobject.
+
+    Examples
+    --------
+
+    .. manim:: PlaneExample
+        :save_last_frame:
+
+        class PlaneExample(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(PI / 3, 100 * DEGREES)
+                ax = ThreeDAxes()
+                normal = ax.c2p(1, 1, 3)
+                normal_vector = Arrow3D(ORIGIN, normal, color=RED)
+                dot = Dot3D(ax.c2p(-2, -1, 1), color=ORANGE)
+                plane = Plane(
+                    point=dot.get_center(),
+                    normal_vect=normal,
+                    u_range=[-2.5, 2.5],
+                    v_range=[-2.5, 2.5],
+                    fill_opacity=0.5,
+                )
+                normal_vector.shift(plane.get_center())
+                self.add(ax, normal_vector, dot, plane)
+                self.wait()
+
+    Parameters
+    ----------
+    point: Sequence[float]
```

Comment:
```suggestion
    point
```
No need for type hints here anymore.